src/ui/app.rs

```diff
@@ -43,6 +43,9 @@ impl App {
                 Menu::Send => ManagerSendState::new(revaultd).into(),
                 // Manager cannot delegate funds, the user is redirected to the home.
                 Menu::DelegateFunds => ManagerHomeState::new(revaultd).into(),
+                Menu::Settings => {
+                    SettingsState::new(self.revaultd.clone().unwrap().config.clone()).into()
```

Comment:
https://github.com/revault/revault-gui/pull/57#discussion_r599412818 is not resolved

Reply:
Sorry!